src/codewhisperer/activation.ts

```diff
@@ -335,8 +335,14 @@ export async function activate(context: ExtContext): Promise<void> {
                 }
             }),
             Commands.register('aws.codeWhisperer.rejectCodeSuggestion', async e => {
-                if (vscode.window.activeTextEditor)
+                if (vscode.window.activeTextEditor) {
                     await InlineCompletion.instance.rejectRecommendation(vscode.window.activeTextEditor)
+                    if (e === 'up') {
+                        await vscode.commands.executeCommand('cursorUp')
+                    } else if (e === 'down') {
```

Comment:
if `e` is some other value that isn't handled here, at least should log `warn()`.

Reply:
Thanks for your suggestion. I added log warn() when the commend argument is unexpected.